### PR TITLE
Update UserGuide to reflect structures better

### PR DESCRIPTION
**Pull request by ExtraShotLatte**
Contributes to #112 again

- Added how to use the examples given in the UserGuide
- Restructure the examples given and their expected actions
- Added Paidlancer logo on the top of page
- Fixed wrong example given in UG where rate was using `p/` instead of the correct `r/`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,18 +3,20 @@ layout: page
...
-Welcome to the Paidlancers User Guide! We're thrilled to have you here! At Paidlancers, we know that freelancing can be both rewarding and challenging.
-As freelancers ourselves, we understand the unique obstacles and opportunities that come with this type of work.
-That's why we created Paidlancers – a desktop app designed to help you streamline your freelancing event management
-tasks.
+![Ui](images/Logo.png)
+
+Welcome to the Paidlancers User Guide! We're thrilled to have you here! At Paidlancers,
+we know that freelancing can be both rewarding and challenging. As freelancers ourselves, we understand the unique
+obstacles and opportunities that come with this type of work. That's why we created Paidlancers – a desktop app
+designed to help you streamline your freelancing event management tasks.
```
Comment (glozxi) [inline docs/UserGuide.md:11]: Nice intro!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -32,14 +34,41 @@ This user guide will serve as a guide to help you get up to speed in no time!
...
+
+**Parameters**
+- Parameters are the words that follow the command word.
+- Parameters are written in UPPERCASE.
+- These include examples such as `INDEX` or `PHONE`.
+- Parameters are meant to be replaced by the user with the relevant information.
+- All parameters are required unless wrapped with `[square brackets]`.
+
+**Example**
+- Examples are shown in the format `commandword + parameters`.
```
Comment (glozxi) [inline docs/UserGuide.md:58]: Perhaps the `+` can be removed?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -150,27 +187,33 @@ Unmarks a specified event in Paidlancers.
...
+**Format**: `newevent n/NAME r/rate a/ADDRESS ds/START_TIME de/END_TIME [t/TAG]…`
 
 - Both `START_TIME` and `END_TIME` must have the format `dd-MM-yyyy HH:mm`.
 
 **Example**:
-- `newevent n/DJ at wedding p/100 a/311, Clementi Ave 2, #02-25 ds/11-03-2023 11:00 de/11-03-2023 17:00 t/friends t/dj`
+- `newevent n/DJ at wedding r/100 a/311, Clementi Ave 2, #02-25 ds/11-03-2023 11:00 de/11-03-2023 17:00 t/friends t/dj`
+  - This command will create a new event named `DJ at wedding` with rate `100` at address `311, Clementi Ave 2, #02-25` from `11-03-2023 11:00` to `11-03-2023 17:00` with tags `friends` and `dj`.
 
 ### List all Events: `listevent` <a id = "list-all-events"></a>
```
Comment (glozxi) [inline docs/UserGuide.md:209]: Perhaps `listevent` could be changed to `list`?

**Review by glozxi — CHANGES_REQUESTED** [review]
Looks good! Just some nits to be fixed :octocat:

File: docs/UserGuide.md (hunk 1 above)
Comment (bryansendeavour) [reply docs/UserGuide.md:11]: Second that

Comment (EksdeeeX) [reply docs/UserGuide.md:11]: Thanks for improving the intro!

File: docs/UserGuide.md (hunk 3 above)
Comment (ExtraShotLatte) [reply docs/UserGuide.md:209]: Done!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -195,26 +239,33 @@ Edits the specified event from the event book.
 - Edits the event at the specified `INDEX`
 
     - The `INDEX` refers to the index number shown in the displayed event list.
-    - `[]` are optional parameters.
     - The `INDEX` must be a positive integer 1, 2, 3, …
+    - `[]` are optional parameters.
     - At least one of the optional fields must be provided.
     - Edits will replace existing values, edits are not cumulative.
     - Tags can be removed by typing `t/` without specifying any tags after it.
```
[the remark this answers is not in the document]
Comment (ExtraShotLatte) [reply docs/UserGuide.md:246]: Good catch! Have updated the UG on this!

**Review by glozxi — APPROVED** [review]
LGTM

**Pull request merged**